docs/ObservabilityPlayground-LandingPage.md

```diff
@@ -0,0 +1,29 @@
+
+![](https://raw.githubusercontent.com/opensearch-project/.github/main/profile/banner.jpg)
+# Observability
+
+### OpenSearch Dashboards
+
+Observability is collection of plugins and applications that let you visualize data-driven events by using Piped Processing Language to explore, discover, and query data stored in OpenSearch.
+Your experience of exploring data might differ, but if you’re new to exploring data to create visualizations, we recommend trying a workflow like the following:
+
+1. Explore data within a certain timeframe using [Piped Processing Language](https://opensearch.org/docs/latest/search-plugins/sql/ppl/index).
+
+2. Use [event analytics](https://opensearch.org/docs/latest/observing-your-data/event-analytics) to turn data-driven events into visualizations.
+![](https://opensearch.org/docs/latest/images/event-analytics.png)
+
+3. Create [operational panels](https://opensearch.org/docs/latest/observing-your-data/operational-panels) and add visualizations to compare data the way you like.
+![](https://opensearch.org/docs/latest/images/operational-panel.png)
+
+4. Use [log analytics](https://opensearch.org/docs/latest/observing-your-data/log-ingestion/) to transform unstructured log data.
```

Comment:
The naming-clash is unfortunate... for now, shouldn't this be https://opensearch.org/docs/latest/observing-your-data/event-analytics/ ? 

Am I correct that you're wanting user-facing documentation for these modules?